go/connector/run.go

```diff
@@ -96,120 +109,28 @@ func Run(
 		// that running `docker logs` to see the output of a connector will not
 		// work. This is acceptable, since all of the stderr output is logged
 		// into the ops collections.
-		"--log-driver",
-		"none",
-	}
-
-	if networkName != "" {
-		imageArgs = append(imageArgs, fmt.Sprintf("--network=%s", networkName))
-	}
-
-	var tempdir, err = ioutil.TempDir("", "connector-files")
-	if err != nil {
-		return fmt.Errorf("creating tempdir: %w", err)
-	}
-	defer os.RemoveAll(tempdir)
-
-	if connectorProxyPath, err := prepareFlowConnectorProxyBinary(tempdir); err != nil {
-		return fmt.Errorf("prepare flow connector proxy binary: %w", err)
-	} else {
-		imageArgs = append(imageArgs,
-			"--entrypoint", connectorProxyPath,
-			"--mount", fmt.Sprintf("type=bind,source=%[1]s,target=%[1]s", connectorProxyPath),
-		)
-	}
-
-	if err := PullRemoteImage(ctx, image, logger); err != nil {
-		// This might be a local image. Log an error and keep going.
-		// If the image does not exist locally, the inspectImage will return an error and terminate the workflow.
-		logger.Log(logrus.InfoLevel, logrus.Fields{
-			"error": err,
-		}, "pull remote image does not succeed.")
-	}
-
-	if inspectOutput, err := InspectImage(ctx, image); err != nil {
-		return fmt.Errorf("inspect image: %w", err)
-	} else {
-		if jsonFiles == nil {
-			jsonFiles = map[string]json.RawMessage{imageInspectJsonFileName: inspectOutput}
-
-		} else {
-			jsonFiles[imageInspectJsonFileName] = inspectOutput
-		}
-	}
-
-	args = append([]string{
-		fmt.Sprintf("--image-inspect-json-path=/tmp/%s", imageInspectJsonFileName),
-		fmt.Sprintf("--log.level=%s", ops.LogrusToFlowLevel(logger.Level()).String()),
+		"--log-driver", "none",
+		// Network to which the container should attach.
+		"--network", networkName,
```

Comment:
I don't think this will work in the case that `networkName` is empty. `--network=` might work OK though?

Reply:
Updated to use `bridge` if networkName is empty. I think this is compatible with your PR as well. Good?

Reply:
Sounds good to me :+1: